examples/scenarioSunlineFiltersCompare.py

```diff
@@ -0,0 +1,352 @@
+
+import numpy as np
+import time
+
+from Basilisk import __path__
+bskPath = __path__[0]
+
+from Basilisk.utilities import SimulationBaseClass, unitTestSupport, macros
+import matplotlib.pyplot as plt
+from Basilisk.utilities import orbitalMotion as om
+from Basilisk.utilities import RigidBodyKinematics as rbk
+
+from Basilisk.simulation import spacecraft, coarseSunSensor
+from Basilisk.fswAlgorithms import sunlineUKF, sunlineSRuKF
+from Basilisk.architecture import messaging
+
+import SunLineKF_test_utilities as Fplot
+
+
+r"""
+Setup 1 - ukF
```

Comment:
:large_orange_diamond: You have comments on the UKF here. Do we need to add some comments for the SRUKF?